app/LoadedApp/LoadedApp.tsx

```diff
@@ -530,7 +532,9 @@ class LoadedAppClass extends Component<LoadedAppClassProps, AppStateLoaded> {
   onClickOKChangeWallet = async () => {
     const { info } = this.state;
     const resultStr =
-      info && info.currencyName !== 'ZEC' ? await this.rpc.changeWalletNoBackup() : await this.rpc.changeWallet();
+      info.currencyName && info.currencyName !== 'ZEC'
+        ? await this.rpc.changeWalletNoBackup()
```

Comment:
This logic looks suspicious to me, it's checking to see if there's a named currency, and if the named currency exists (and isn't ZEC), it calls `changeWalletNoBackup()` instead of `changeWallet()`?? Still, you're not changing the functionality here, just improving the interface, so it's not a blocker.

Reply:
When you change to another wallet, I decided to backup the active wallet, just in case you want to come back again to the prior wallet... and  decided to use the backup functionality only for `mainnet`. If `info.currencyName !== 'ZEC'` is some kind of test server, and in this case I change to another wallet without any backup.
Maybe this decision is not totally correct, What do you think?